sensenet/models/wrappers.py

```diff
@@ -48,6 +45,7 @@ def __call__(self, input_data):
 
 class ObjectDetector(object):
     def __init__(self, model, settings):
+        self._unfiltered = settings.output_unfiltered_boxes
```

Comment:
When creating the model with 

`wrappers.create_model(meta, settings=SENSENET_CONFIG)`

settings is a dict, so it raises an error:

`AttributeError: 'dict' object has no attribute 'output_unfiltered_boxes'`

I was able to run the conversion changing this line to 

`self._unfiltered = settings["output_unfiltered_boxes"]`

but it won't work when an actual settings object is received...

Reply:
@unmonoqueteclea - I fixed this so you can provide either thing.  Can you check and if it works approve the PR?